src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1965,6 +2081,7 @@
   "com.minecolonies.research.combat.flee.name": "Flee",
   "com.minecolonies.research.combat.flee.subtitle": "Sometimes it's better just to run.",
   "com.minecolonies.research.combat.hotfoot.name": "Hotfoot",
+  "com.minecolonies.research.combat.taunt.name": "Taunt",
   "com.minecolonies.research.combat.taunt.subtitle": "Your mother was a hamster and your father smelt of elderberries!",
   "com.minecolonies.research.combat.arrowuse.name": "Consume Arrows",
   "com.minecolonies.research.combat.arrowuse.subtitle": "They work better with ammo.",
```

Comment:
The research strings should no longer appear in the source en_us.json when they are being added by datagen.  While this shouldn't cause an actual bug (provided that the datagen is smart enough to replace existing values rather than simply appending), it will cause confusion down the line when someone tries to edit the wrong string and is confused when their changes do not take effect.

Conversely, however, the generated en_us.json and other files that are the result of runData should also be included in this commit.

Reply:
Unless the change to this file is because you are making the datagen directly write to this file instead of generating a replacement in the generated directory instead.  In which case: don't do that.  Revert the changes to this file, then delete the research strings, then change the output directory of the datagen.

Reply:
Yes, right now it's feeding back into the same file as the origin.  I understand it's possible to produce a separate file, but doing so will result in two possible sources-of-true, in violation of the DRY rule, and especially with how many people you've got working on this project, that's a lot of potential for incident or frustration.  Is there a particular benefit to your approach, or is there a reason that Structurize wasn't doing it this way?

Reply:
As discussed on Discord, putting generated data into a non-generated folder is worse, because people will see "oh, here's a file in the hand-editing directory, let me change it there" and then their changes mysteriously vanish because they happened to be values that the datagen replaced.  With the way I suggested, at least there's the context that everything in the "generated" folder will get replaced on rundata, so you should never hand-edit them.  It's more consistent.